tlslite/keyexchange.py

```diff
@@ -242,20 +242,19 @@ def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
     # 2048-bit MODP Group (RFC 3526, Section 3)
     dh_g, dh_p = goodGroupParameters[2]
 
-    # RFC 3526, Section 8.
-    strength = 160
-
-    def makeServerKeyExchange(self, sigHash=None):
-        """Prepare server side of key exchange with selected parameters"""
+    def makeServerKeyExchange(self):
+        """
+        Prepare server side of anonymous key exchange with selected parameters
+        """
         # Per RFC 3526, Section 1, the exponent should have double the entropy
         # of the strength of the curve.
-        self.dh_Xs = bytesToNumber(getRandomBytes(self.strength * 2 // 8))
+        self.dh_Xs = bytesToNumber(getRandomBytes(32))
```

Comment:
"32" is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) so it probably would be better to retain `self.strength`